js/forms.js

```diff
@@ -279,85 +279,181 @@
+        ){
+          highlightElements(data, $input, $autocomplete);
+        }
+      }
+
+      // function to displaying elements
+      function highlightElements(data, $input, $autocomplete){
+        // Check if the input isn't empty and isn't equal to the old option
+        if (val !== '' && currentAutocompleteVal !== val) {
+          var val = $input.val().toLowerCase(),
+          maxElementsAmount = options.maxElementsAmount;
+          $autocomplete.empty();
+
+          for (var key in data) {
+
+            if (maxElementsAmount === 0) {
+              break;
+            }
+
+            if (data.hasOwnProperty(key) &&
+            key.toLowerCase().indexOf(val) !== -1 &&
+            key.toLowerCase() !== val) {
+              if (maxElementsAmount !== null) {
+                maxElementsAmount--;
+              }
+
+              autocompleteOption = $('<li></li>');
+              if (!!data[key]) {
+                autocompleteOption.append('<img src="' + data[key] + '" class="right circle"><span>' + key + '</span>');
+              } else {
+                autocompleteOption.append('<span>' + key + '</span>');
+              }
+              $autocomplete.append(autocompleteOption);
+
+              highlight(val, autocompleteOption);
+            }
+          }
 
-        // Check if data isn't empty
-        if (!$.isEmptyObject(data)) {
-          // Create autocomplete element
+          currentAutocompleteVal = val;
+          autocompleteOption = $autocomplete.find('li');
+          currentLi = 0;
+          $(autocompleteOption[currentLi]).addClass('active');
+        }
+      }
+
+      // arrow usage - on key down
+      function arrowUsage(e, $autocomplete) {
+        // Capture up and down key
+        if (
+        e.which === 38
+        || e.which === 40
+        ) {
+          e.preventDefault();
+
+          if (currentLi > 0 && e.which === 38) {
+            currentLi--;
+          }
+
+          if (currentLi < (autocompleteOption.length - 1) && e.which === 40) {
+            currentLi++;
+          }
+
+          $autocomplete.find('.active').removeClass('active');
+          $(autocompleteOption[currentLi]).addClass('active');
+        }
+      }
+
+      // reset current element position
+      function resetCurrentElement($autocomplete) {
+        currentLi = 0;
+        $autocomplete.find('.active').removeClass('active');
+        $(autocompleteOption[currentLi]).addClass('active');
+      }
+
+
+      // initialization for each element
+      return this.each(function () {
+        var $input = $(this),
+        data = options.data,
+        $inputDiv = $input.closest('.input-field');
+
+        // remove list if it exist
+        if($inputDiv.find('.autocomplete-content').length !== 0){
+          $inputDiv.find('.autocomplete-content').remove();
+        }
+
+        // Check if data isn't empty and append autocomplete element if doesn't exist yet - run only once
+        if(
```

Comment:
if statement line breaks and indenting look weird. (Not normal JavaScript style)